.github/workflows/operator_ci.yaml

```diff
@@ -0,0 +1,184 @@
+        with:
+          username: 'csiblock1'
+          password: '${{ secrets.CSIBLOCK_DOCKERHUB_PASSWORD }}'
+      - name: Cache Docker layers
+        uses: actions/cache@v2
+        with:
+          path: /tmp/.buildx-${{ matrix.image_type }}
+          key: ${{ runner.os }}-buildx-${{ github.sha }}
+          restore-keys: |
+            ${{ runner.os }}-buildx-
+      - name: Build and push ${{ matrix.image_type }}
+        uses: docker/build-push-action@v2
+        with:
+          context: .
+          platforms: linux/amd64,linux/ppc64le,linux/s390x
+          push: true
+          tags: |
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:${{ needs.prepare_env.outputs.operator_specific_tag_for_test }}
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:${{ needs.prepare_env.outputs.image_branch_tag }}
+            csiblock1/ibm-block-csi-${{ matrix.image_type }}:latest
+          file: build/Dockerfile.${{ matrix.image_type }}
+          cache-from: type=local,src=/tmp/.buildx-${{ matrix.image_type }}
+          cache-to: type=local,dest=/tmp/.buildx-new-${{ matrix.image_type }}
+        # Temp fix
+        # https://github.com/docker/build-push-action/issues/252
+        # https://github.com/moby/buildkit/issues/1896
+      - name: Move cache
+        run: |
+          [[ -G /tmp/.buildx-${{ matrix.image_type }} ]] && rm -rf /tmp/.buildx-${{ matrix.image_type }}
+          mv /tmp/.buildx-new-${{ matrix.image_type }} /tmp/.buildx-${{ matrix.image_type }}
+    outputs:
+      operator_specific_tag_for_test: "${{ needs.prepare_env.outputs.operator_specific_tag_for_test }}"
+
+  create_cluster:
+    runs-on: ubuntu-latest
+    name: "Create cluster"
+    needs: csi_build_and_push_images
+    env:
+      csiblock_dockerhub_password: ${{ secrets.CSIBLOCK_DOCKERHUB_PASSWORD }}
+      controller_repository_for_test: csiblock1/ibm-block-csi-controller
+      node_repository_for_test: csiblock1/ibm-block-csi-node
+      operator_image_repository_for_test: csiblock1/ibm-block-csi-operator
+      operator_specific_tag_for_test: "${{ needs.csi_build_and_push_images.outputs.operator_specific_tag_for_test }}"
+      operator_image_for_test: "csiblock1/ibm-block-csi-operator:${{ needs.csi_build_and_push_images.outputs.operator_specific_tag_for_test }}"
+    timeout-minutes: 7
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+      - uses: FranzDiebold/github-env-vars-action@v2
+      - name: Retrieve ci dependencies
+        uses: actions/download-artifact@v2
+        with:
+          name: ci-dependencies
+          path: /home/runner
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: '3.x'
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.pythonLocation }}
+          key: ${{ env.pythonLocation }}-${{ hashFiles('dev-requirements.txt') }}
+      - name: Get driver branch
+        id: driver_branch
```

Comment:
branch, or branch tag?

Reply:
I still see `Get driver branch`, is this intentional?

Reply:
no I changed it now